usr/src/test/os-tests/runfiles/default.run

```diff
@@ -107,17 +103,6 @@ arch = i86pc
 timeout = 300
 tests = ['ldt', 'badseg']
 
-[/opt/os-tests/tests/imc_test]
```

Comment:
This file needs copyright update

Reply:
Because this was a mismerge, I didn't think it did.